design/scheduled-update-check.md

```diff
@@ -0,0 +1,66 @@
+# Scheduled Update Checks for Online Installs
+
+KOTS (Admin Console) does not automatically check for new updates. This proposal adds the 
+ability for KOTS (Admin Console) to check for updates every X period of time for online installations.
+
+## Goals
+
+- Pull new updates automatically for KOTS (Admin Console) online installations
+
+## Non Goals
+
+- Deployment of the new updates
+- Scheduled Update Checks for Airgap Installations
+
+## Background
+
+Many customers assume that scheduled update checks are a given and are surprised when they do not receive 
+updates automatically.
+
+## High-Level Design
+
+1. This proposal is only for online installations in KOTS Admin console.
+2. The default period of time (X) is configurable.
+3. The ability to disable this feature completely.
+
+The update check will happen in a background process/thread which is triggered every X period of time.
+On each trigger, the process will get all available applications and run update checks for each app.
+The update check will automatically create new versions for each update available in an app.
+
+## Detailed Design
+
+3 new columns (`update_checker_enabled` & `update_checker_interval` & `update_checker_status`) will be 
+added to the `app` table in the postgress database using a migration.
+
+* The `update_checker_enabled` column is a bool and defaults to `true`. 
+* The `update_checker_interval` column is an integer and defaults to `360` minutes.
```

Comment:
I think a Go duration would be more compatible (we use this for snapshots).  So `6h` would be the default?

Reply:
yes, good point, i'll update